utility/src/main/java/org/oppia/android/util/logging/performancemetrics/PerformanceMetricsAssessorImpl.kt

```diff
@@ -70,4 +78,21 @@ class PerformanceMetricsAssessorImpl @Inject constructor(
       else -> HIGH_MEMORY_TIER
     }
   }
+
+  override fun getCpuUsage(): Double {
```

Comment:
When actually logging this, I suppose we'd want to measure it within a specific time window--right? Otherwise we're logging a running average rather than the CPU utilization at specific points.

I could see this being "getCurrentCpuUsageSinceAppStart" and then using the worker to call this before and after a specific time window, and then subtract to compute the total time. That being said, I think this would need to be updated to return the CPU execution time rather than usage since usage needs to be re-computed relative to the time window.